src/scss/_buttons.scss

```diff
@@ -32,7 +32,7 @@
         }
     }
 
-    &:disabled {
```

Comment:
:disabled is a html attribute and .disabled is a class, are you sure you want to swap one for the other ? maybe you would like both ?